neqo-transport/src/pmtud.rs

```diff
@@ -604,31 +623,50 @@ mod tests {
         assert_eq!(expected_lc, pmtud.loss_counts);
 
         // A packet of size 5000 was lost, which should increase loss counts >= 5000 by one. There
-        // have now been MAX_PROBES losses of packets >= 5000, so the PMTUD process should have
-        // restarted.
+        // have now been MAX_PROBES losses of packets >= 5000. That is larger than the current
+        // MTU, so nothing will happen.
+        expected_lc = search_table_inc(&pmtud, &expected_lc, 5000);
         pmtud.on_packets_lost(&[make_sentpacket(0, now, 5000)], &mut stats, now);
-        assert_pmtud_restarted(&pmtud);
-        expected_lc.fill(0); // Reset the expected loss counts.
+        assert_eq!(expected_lc, pmtud.loss_counts);
 
-        // Two packets of size 4000 were lost, which should increase loss counts >= 4000 by two.
-        let expected_lc = search_table_inc(&pmtud, &expected_lc, 4000);
-        let expected_lc = search_table_inc(&pmtud, &expected_lc, 4000);
+        // Two packets of size 4000 were lost, which should increase loss counts >= 4000 by one.
```

Comment:
No, the comment is correct. We do only increase by one of several packets of the same size are declared lost on the same ACK. Otherwise each cwnd overshoot would make us reprobe the MTU.